window_managers/hyprland/install-hyprland.sh

```
@@ -0,0 +1,533 @@
+        log_warning "No AUR helper found. Consider installing yay or paru for additional packages."
+    fi
+    
+    log_success "Hyprland packages installed"
+}
+
+install_hyprland_fedora() {
+    log_info "Installing Hyprland and Wayland packages on Fedora..."
+    
+    # Enable RPM Fusion repositories
+    sudo dnf install -y \
+        https://download1.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm \
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)